### PR TITLE
Fix change output when are used all balance

### PR DESCRIPTION
**Pull request by ronaldsg20**
I've updated the pegin-tx controller in order to check
                if the change output has zero amount, in that case ot
                  wot be on the returned normalizedTx

### CODE REVIEW

File: src/controllers/pegin-tx.controller.ts
Comment (josedahlquist) [inline src/controllers/pegin-tx.controller.ts:70]: as @jeremy-then mentioned. We should be returning the value.
On top of that we should always enclose conditions/loops in curly braces
```suggestion
          if (inputsAmount.amount - (createPeginTxData.amountToTransferInSatoshi + fee) < 0) {
            return reject(new Error('The stored input list is has not enough amount'));
          }
```

**Review by josedahlquist — APPROVED** [review]
leaving my approval although the comments needs to be attended.
Please don't merge until @jeremy-then approves the PR

**Review by jeremy-then — APPROVED** [review]
LGTM.

**Pull request merged**
